src/pages/tech_stack.adoc

```diff
@@ -22,7 +22,8 @@ should take place, we use data to drive every aspect of the business.
 🆓 *Open source*
 
 We strongly believe in open sourcing our tech. But don't take our word
-for it. Just check out our http://picnic.tech/[Github page].
+for it. Just check out our http://picnic.tech/[OSS library] or go directly
```

Comment:
```suggestion
for it. Just check out our http://picnic.tech/[OSS libraries] or go directly
```

Reply:
The way I mean it "The library of the Open Source Software". Do you think I should change it?

Reply:
Ahhhh. That makes sense, but I suspect a subset of readers may also interpret it the way I did. Maybe we can use a synonym. "Overview", "listing", ... :thinking: